hack/FRAMEWORK_BUILD_VERSION

```diff
@@ -1 +1 @@
-v0.17.0
+v0.10.1
```

Comment:
Should this be `v0.11.1`?

Reply:
Hmm, yes. Could have sworn I changed that. Updating now!